concourse/tasks/delete_admin.yml

```diff
@@ -0,0 +1,25 @@
+---
+platform: linux
+image: docker:///governmentpaas/cf-uaac
+inputs:
+  - name: paas-cf
+  - name: cf-secrets
+  - name: cf-manifest
+  - name: admin-creds
+run:
+  path: sh
+  args:
+    - -e
+    - -c
+    - |
+      NAME=$(cat admin-creds/username)
+
+      VAL_FROM_YAML=$(pwd)/paas-cf/concourse/scripts/val_from_yaml.rb
+      UAA_ADMIN_CLIENT_PASS=$($VAL_FROM_YAML secrets.uaa_admin_client_secret cf-secrets/cf-secrets.yml)
+      UAA_ENDPOINT=$($VAL_FROM_YAML properties.uaa.url cf-manifest/cf-manifest.yml)
+
+      uaac target "${UAA_ENDPOINT}" --skip-ssl-validation
```

Comment:
Can we add `./paas-cf/concourse/scripts/import_bosh_ca.sh` before this and remove `--skip-ssl-validation` like df2f0803f0f3a573d99d36e229110855da4398cd?